diesel/src/sqlite/connection/mod.rs

```diff
@@ -54,6 +54,10 @@ impl Connection for SqliteConnection {
     type Backend = Sqlite;
     type TransactionManager = AnsiTransactionManager;
 
+    /// Establish a connection to `database_url`.
```

Comment:
```suggestion
    /// Establish a connection to the `database_url`.
```

Reply:
```suggestion
    /// Establish a connection to the database specified by `database_url`.
    ///
    /// Supported connection URLs are:
    ///
    /// - File paths (`test.db`)
    /// - [URIs](https://sqlite.org/uri.html) (`file://test.db`)
    /// - Special identifiers (`:memory:`)
    ////
```

(Just including the information from the type itself here as well)

Reply:
Thanks for your suggestions. I've updated the diff, and to avoid repeating the same info, I've included a link to `SqliteConnection` https://github.com/diesel-rs/diesel/blob/master/diesel/src/sqlite/connection/mod.rs#L29-L36